benchmarks/AutomaticDifferentiation/JuliaAD.jmd

```diff
@@ -0,0 +1,33 @@
+---
+title: Julia AD Benchmarks
+author: Chris Rackauckas

```

Comment:
Strictly speaking I'm the author of that code ;)

Reply:
Feel free to edit it in a PR